### ISSUE TITLE
Demo MODIS-TOA for both downstream task and reconstruction loss

### ISSUE THREAD

**Issue by jordancaraballo** [nasa-nccs-hpda/pytorch-caney#35]
Demo MODIS-TOA for both downstream task and reconstruction loss

I already have the reconstruction notebook from satvision-sr, I can just push it. We will need a demo dataloader on the production branch to support satvision-toa.

**Comment by cssprad1** [comment]
Consider done with #63
